contents/blog/how-to-measure-product-market-fit.md

```diff
@@ -0,0 +1,352 @@
+
+> Show `unique users` who performed a `discovery` for the first time in the last `12 months` and then came back to perform a `discovery` on any of the next months.
+
+It's also a good idea to track different cohorts, such as retention for users who match your ideal customer profile, or split by job title. This will help you understand how strong your fit is for different groups, and plan your roadmap accordingly.
+
+> 💡 **PostHog Tip:** It's easy to track retention in PostHog using the [retention insight type](/docs/product-analytics/retention), which generates both a retention curve and a cohort retention table. You can apply different cohorts (e.g. users by job title, age, etc.) to segment your retention for specific groups, which is a useful way to compare the strength of your product-market fit for different types of users. See also: [How to discover features that drive user retention](/tutorials/feature-retention).
+
+### When to use retention
+
+Retention is another non-negotiable way to measure product-market fit. It's simplest lagging indicator of product-market fit, so there's really no reason not to use it.
+
+| **Pros**                                                                                                                 | **Cons**                                                                                                                 |
+|--------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------| 
+| <span className="text-green text-lg">✔</span> Reliable lagging indicator of product-market fit                           | <span className="text-red text-lg">✖</span> Not predictive of revenue – will users actually pay for your product?        |
+| <span className="text-green text-lg">✔</span> Can be broken down by cohort for comparison                                | <span className="text-red text-lg">✖</span> You need several weeks or months of data to get a reliable signal            |
+| <span className="text-green text-lg">✔</span> Easy to understand – flat is good, not flat is bad!                        | <span className="text-red text-lg">✖</span> Won't help you understand why you do or don't have product-market fit        |
+
+## Indicator #5: The burn multiple
+
+- **Type:** Lagging indicator
+- **Good for:** Late-stage companies seeking investment
+
+Created by former PayPal COO David Sacks, burn multiple measures how much you're spending to grow revenue. The less you spend, the more efficient your growth. More efficient growth = stronger product-market fit.
+
+Because it measures efficiency rather than product-market fit directly, it's best used as a way to prove how _strong_ your product-market fit is. For this reason, it's especially useful for later-stage startups seeking further investment.
```

Comment:
```suggestion
Because it measures efficiency rather than product-market fit directly, it's best used as a way to prove how _strong_ your product-market fit is. It's especially useful for later-stage startups seeking further investment, but it's a board-level metric – i.e. product teams won't find it useful.
```